### PR TITLE
simplewallet: report timestamp based expected unlock time on balance

### PR DESCRIPTION
**Pull request by moneromooo-monero**

### CODE REVIEW

**Comment by selsta** [comment]
```
2020-04-19T16:21:45.7301356Z /home/runner/work/monero/monero/src/wallet/wallet_rpc_server.cpp: In member function ‘bool tools::wallet_rpc_server::on_getbalance(const request&, tools::wallet_rpc::COMMAND_RPC_GET_BALANCE::response&, epee::json_rpc::error&, const connection_context*)’:
2020-04-19T16:21:46.6547656Z /home/runner/work/monero/monero/src/wallet/wallet_rpc_server.cpp:440:139: error: no match for ‘operator=’ (operand types are ‘std::map<unsigned int, std::map<unsigned int, std::pair<long unsigned int, long unsigned int> > >::mapped_type {aka std::map<unsigned int, std::pair<long unsigned int, long unsigned int> >}’ and ‘std::map<unsigned int, std::pair<long unsigned int, std::pair<long unsigned int, long unsigned int> > >’)
2020-04-19T16:21:46.6553748Z            unlocked_balance_per_subaddress_per_account[account_index] = m_wallet->unlocked_balance_per_subaddress(account_index, req.strict);
```

File: src/simplewallet/simplewallet.cpp [hunk 1]
```diff
@@ -6375,7 +6380,7 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
     if (locked_blocks > 1000000)
     {
       fail_msg_writer() << tr("Locked blocks too high, max 1000000 (˜4 yrs)");
-      return false;
+      //return false;
```
Comment (selsta) [inline src/simplewallet/simplewallet.cpp:6383]: Is this intended?

Comment (moneromooo-monero) [reply src/simplewallet/simplewallet.cpp:6383]: No, it was for testing -_-

File: src/wallet/wallet_rpc_server_commands_defs.h [hunk 2]
```diff
@@ -84,6 +84,7 @@ namespace wallet_rpc
       std::string label;
       uint64_t num_unspent_outputs;
       uint64_t blocks_to_unlock;
+      uint64_t time_to_unlock;
```
Comment (selsta) [inline src/wallet/wallet_rpc_server_commands_defs.h:87]: Does this require `WALLET_RPC_VERSION_MINOR` bump?

**Pull request merged**